### PR TITLE
docs: Updated crypt/cryptdecode docs with additional info

### PR DESCRIPTION
**Pull request by eharris**
Add additional links to crypt/cryptdecode docs.

### CODE REVIEW

File: docs/content/commands/rclone_cryptdecode.md [hunk 1]
```diff
@@ -40,4 +40,5 @@ See the [global flags page](/flags/) for global options not listed here.
 ## SEE ALSO
 
 * [rclone](/commands/rclone/)	 - Show help for rclone commands, flags and backends.
+* [crypt](/crypt/)	 - Info about the crypt overlay
```
Comment (ncw) [inline docs/content/commands/rclone_cryptdecode.md:43]: This is an auto generated file from `rclone cryptdecode --help` so I don't think you can patch it like that alas.

You could put a link in the docs in `cmd/cryptdecode/cryptdecode.go`

Comment (eharris) [reply docs/content/commands/rclone_cryptdecode.md:43]: The source code doesn't appear to contain the `## SEE ALSO` portion of the docs.  How do I add a customized item to that?

Comment (ncw) [reply docs/content/commands/rclone_cryptdecode.md:43]: That is created by cobra I think. I'm not 100% sure how you do that!

I suggest just adding a note at the bottom of the main text.

**Comment by ivandeex** [comment]
@ncw Please review

**Review by ncw — APPROVED** [review]
That looks great - thank you

I'll merge it now

(Sorry for the delay!)

**Pull request merged**